manifests/kustomize/base/webhook/kustomization.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - pipelineversion-cert-issuer.yaml
+  - pipelineversion-cert.yaml
+  - validating-pipelineversion.yaml
+commonLabels:
+  app: pipeline-version-cert-manager
```

`commonLabels` is deprecated.
Can we use the following instead?

```suggestion
labels:
  - pairs:
      app: pipeline-version-cert-manager
```